src/dxvk/dxvk_context.h

```diff
@@ -87,10 +87,23 @@ namespace dxvk {
      * \param [in] targets Render targets to bind
      */
     void bindRenderTargets(
-            DxvkRenderTargets&&   targets) {
+            DxvkRenderTargets&&   targets,
+            bool                  hazardColor = false,
+            bool                  hazardDepth = false) {
```

Comment:
I suppose the plan is to not transition any attachment that's in `FEEDBACK_LOOP_OPTIMAL` back to `ATTACHMENT_OPTIMAL` at any point?

If so, keeping the function argument is fine, but this should really be something like `VkImageAspectFlags feedbackLoop` or something.

Also, please no default argument values here.